Packs/Barikat-ClosedQradarAndXsoar/Scripts/BClosedQradarAndXsoar/BClosedQradarAndXsoar.yml

```diff
@@ -0,0 +1,22 @@
+args:
+- name: note
+  type: textArea
```

Comment:
please add `description`
description: X

Reply:
description: The note entered by the analyst is recorded in both Qradar offense and Xsoar. At the moment, a single note is being written in two places.